src/python/grpcio/grpc/_server.py

```diff
@@ -361,9 +363,9 @@ def abort(self, code: grpc.StatusCode, details: str) -> None:
             self._state.aborted = True
             raise Exception()
 
-    def abort_with_status(self, status: grpc.Status) -> None:
+    def abort_with_status(self, status: grpc.Status) -> NoReturn:
         self._state.trailing_metadata = status.trailing_metadata
-        self.abort(status.code, status.details)
+        self.abort(status.code, status.details)  # pytype: disable=bad-return-type
```

Comment:
Pytype is complaining here return should be `None`, but since `abort` always throws error, we don't expect any return here.